beta-src/src/state/game/game-api-slice.ts

```diff
@@ -325,17 +333,53 @@ const drawBuilds = (state) => {
   }
 };
 
+const updateUnitsRetreat = (state) => {
```

Comment:
can you move this to its own file and just import it.

Reply:
Im not sure this would be ideal because this function is setting many commands and the only file that does this appears to be game-api-slice.